src/components/Modals/SendTipModal.tsx

```diff
@@ -275,19 +280,13 @@ const { openModal, Modal } = createContextModal<{
             {amount === '-1' && (
               <InputNumber
                 name="customAmount"
-                placeholder="Your tip"
+                placeholder="Your tip. Minimum 1000 BUZZ"
                 variant="filled"
-                icon={<IconBolt size={18} fill="currentColor" />}
                 rightSectionWidth="10%"
-                min={1}
+                min={1000}
```

Comment:
I'm not sure if we should set a strict min here, having it open would let people tip below the preset values or get really specific like `99`

Reply:
This was a business request 😬 mainly to avoid bulling and whatnot hopefully.